.idea/misc.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This `.idea` file must be ignored if already sync with master, cmiiw mas @pandubaraja

Reply:
Let me try wait for a second